src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCashAdvanceServiceImpl.java

```diff
@@ -70,5 +65,40 @@ public ConcurAccountInfo findAccountingInfoForCashAdvanceLine(ConcurStandardAcco
         }
         return info;
     }
+    
+    private boolean cashAdvanceKeyExists(ConcurStandardAccountingExtractDetailLine line) {
+        return StringUtils.isNotBlank(line.getCashAdvanceKey());
+    }
+    
+    @Override
+    public boolean isPreTripCashAdvanceRequestLine(ConcurStandardAccountingExtractDetailLine saeLine) {
+        if (StringUtils.isNotEmpty(saeLine.getCashAdvancePaymentCodeName()) 
+                && StringUtils.equalsIgnoreCase(saeLine.getCashAdvancePaymentCodeName(), ConcurConstants.SAE_REQUESTED_CASH_ADVANCE_INDICATOR)) {
+            return true;
```

Comment:
There seem to be a few new methods in this PR where you could just return the value of the condition directly, so that you don't have to keep writing: "if (condition) then return true, else return false"

Reply:
My intent was complete clarity of what the code was doing because of the fact we have "cash advance request" and "cash advance reimbursement" processing now being intertwined in the same file(s) with existing code for both just referring to "cash advance" where changes have not been made or are needed yet.
Please note that the conditions to determine a cash advance are different for a request vs reimbursement with different data fields being needed.